avocado/core/test.py

```diff
@@ -871,3 +871,7 @@ def __init__(self, *args, **kwargs):
 
     def test(self):
         self.error(self.exception)
+
+
+class DummyTest(object):
```

Comment:
I think it'd be more useful to either directly use `SkipTest`, or inherit `DummyTest` from `SkipTest` as this one would fail to push early status:

```
[medic@w541 avocado 2]$ avocado run --loaders dummy -- asdfksadlfa
JOB ID     : 68e31b048d48f47e62a7a5c30534f422154bc65b
JOB LOG    : /home/medic/avocado/job-results/job-2016-10-26T16.04-68e31b0/job.log
TESTS      : 1

Avocado crashed: TestError: Process died before it pushed early test_status.
Traceback (most recent call last):

  File "/home/medic/Work/Projekty/avocado/avocado/avocado/core/job.py", line 471, in run_tests
    replay_map)

  File "/home/medic/Work/Projekty/avocado/avocado/avocado/core/runner.py", line 538, in run_suite
    summary, deadline)

  File "/home/medic/Work/Projekty/avocado/avocado/avocado/core/runner.py", line 364, in run_test
    test_status.wait_for_early_status(proc, 10)

  File "/home/medic/Work/Projekty/avocado/avocado/avocado/core/runner.py", line 150, in wait_for_early_status
    raise exceptions.TestError("Process died before it pushed "

TestError: Process died before it pushed early test_status.

Please include the traceback info and command line used on your bug report
Report bugs visiting https://github.com/avocado-framework/avocado/issues/new
JOB HTML   : /home/medic/avocado/job-results/job-2016-10-26T16.04-68e31b0/html/results.html
```
vs.
```
[medic@w541 avocado 4]$ avocado run --loaders dummy -- asdfksadlfa
JOB ID     : 0b5fec283d343c509b7a6147d1fbd774dafaf923
JOB LOG    : /home/medic/avocado/job-results/job-2016-10-26T16.04-0b5fec2/job.log
TESTS      : 1
 (1/1) asdfksadlfa: SKIP
RESULTS    : PASS 0 | ERROR 0 | FAIL 0 | SKIP 1 | WARN 0 | INTERRUPT 0
TESTS TIME : 0.00 s
JOB HTML   : /home/medic/avocado/job-results/job-2016-10-26T16.04-0b5fec2/html/results.html
```